charts/refarch-templates/templates/configmap.yaml

```
@@ -1,6 +1,6 @@
 {{- $dot := . }}
 {{- range $configmap := .Values.configMaps -}}
-{{ $data := dict "dot" $dot "module" $configmap }}
+{{ $data := dict "dot" $dot "module" $configmap "moduleName" $configmap.name }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)